java/GenericsZodachiOne.java

```diff
@@ -0,0 +1,19 @@
+//Уровень 1. Напишите простую универсальную версию метода isEqualTo, которая сравнивает два его аргумента с методом equals и возвращает true,
+// если они равны, и false в противном случае. Используйте этот универсальный метод в программе, которая вызывает isEqualTo с различными
+// встроенными типами,такие как объект или целое число. Какой результат вы получите при попытке запустить эту программу?
+
+public class GenericsZodachiOne {
+
+    public static void main(String[] args) {
+
+
+
+    }
+}
+
+class compare {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).